generators/server/templates/src/main/java/package/config/CacheConfiguration.java.ejs

```diff
@@ -753,18 +753,17 @@ public class CacheConfiguration<% if (cacheProvider === 'hazelcast') { %> implem
     <%_ } _%>
 
     <%_ if (cacheProvider === 'redis') { _%>
-    private final javax.cache.configuration.Configuration<Object, Object> jcacheConfiguration;
-
-    public CacheConfiguration(JHipsterProperties jHipsterProperties) {
+    @Bean
+    @DependsOn({"testContainerRedis"})
```

Comment:
I don't understand why the production configuration should depend on the testContainer configuration or do I miss anything?

Reply:
Yes you are totally right ! I am still working on it, I need to rework on this part